packages/editor/src/components/document-bar/style.scss

```diff
@@ -25,10 +25,6 @@
 			background: $gray-200;
 		}
 	}
-
-	@include break-large() {
-		width: min(100%, 450px);
-	}
```

Comment:
I'm not sure just why the width needed a distinct maximum at the large breakpoint. I moved the `450px` up to the base style on L13 and with the overall changes the Document Bar smoothly narrows according to available space.